include/swift/SIL/BridgedSwiftObject.h

```diff
@@ -0,0 +1,42 @@
+//===--- BridgedSwiftObject.h - C header which defines SwiftObject --------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// This is a C header, which defines the SwiftObject header. For the C++ version
+// see SwiftObjectHeader.h.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef SWIFT_SIL_BRIDGEDSWIFTOBJECT_H
+#define SWIFT_SIL_BRIDGEDSWIFTOBJECT_H
+
+#include <stdint.h>
+
+#if !__has_feature(nullability)
```

Comment:
I am fine fixing this in a follow up, but isn't __has_feature an extension? @compnerd is my memory right? I think we may need to do something more here.

Reply:
This is handled in type_traits.h, where __has_feature is defined to 0 if it's not available. Also, it's tested with the windows PR testing - so it works for windows.